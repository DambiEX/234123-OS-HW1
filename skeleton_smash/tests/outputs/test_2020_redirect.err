smash error: kill: invalid arguments
smash error: kill: invalid arguments
smash error: chmod: invalid aruments
