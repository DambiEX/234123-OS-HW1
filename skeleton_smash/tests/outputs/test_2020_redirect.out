smash> smash> smash> smash> smash> test_redirect
smash> smash> again_redirect
smash> smash> again_redirect
test_redirect
smash> smash> smash>  pid is 5354
smash> smash> smash>  pid is 5354
smash>  pid is 5354
smash> smash> smash>  pid is 5354
smash>  pid is 5354
smash> mypid
new
smash> smash> smash> [1] sleep 100& 
smash> smash> smash> smash> smash> k
mypid
new
s
t
smash> smash> k
mypid
new
p
s
t
smash> k
mypid
new
p
s
t
smash> k
mypid
new
p
s
t
smash> k
mypid
new
p
s
t
smash> before
smash> again_redirect
test_redirect
smash> smash> smash> smash> k
mypid
p
s
t
smash> smash> k
mypid
new
p
s
t
smash> k
mypid
new
p
s
t
smash> smash> smash> k
mypid
new
p
s
t
smash> k
mypid
new
p
s
t
smash> smash> smash> k
mypid
new
p
s
t
smash> smash> smash> k
mypid
new
p
s
t
test
smash> smash> 0
2
end: 4
smash> smash> smash> smash: sending SIGKILL signal to 0 jobs:
